### PR TITLE
expand SSL guide

### PR DESCRIPTION
**Pull request by indirect**

### CODE REVIEW

File: source/v1.15/guides/rubygems_tls_ssl_troubleshooting_guide.html.md [hunk 1]
```diff
@@ -1,40 +1,159 @@
...
+    - [Still having trouble?]()
+    - [Contributing to this guide]()
+
+## The Problems
+
+### Why am I seeing `certificate verify failed`?
+
+This error happens when your computer is missing a file that it needs to verify that the server behind RubyGems.org is the correct one.
+The latest version of RubyGems should fix this problem, so we recommend updating to the current version. To tell RubyGems to update itself to the latest version, run `gem update --system`. If that doesn’t work, try the manual update process below.
+(What do we mean by updating “should fix this problem”? Review our article on how SSL works with RubyGems to gain a better understanding of the underlying problems.
```
Comment (segiddins) [inline source/v1.15/guides/rubygems_tls_ssl_troubleshooting_guide.html.md]: missing a closing `)`

File: source/v1.15/guides/rubygems_tls_ssl_troubleshooting_guide.html.md [hunk 2]
```diff
@@ -26,312 +30,408 @@ If you're not interested in the reasons, and just want to get things fixed as qu
...
-1. The command(s) you are running
+### Creating an issue
+[creating-an-issue]: #creating-an-issue
 
-2. The output from that command
+If none of these instructions fixed the problem, the next step is to open an issue.
 
-3. The output from running `gem env`
+(Create an issue in the [RubyGems issue tracker](https://github.com/rubygems/rubygems/issues)
+If your error came from `gem install`. If it came from `bundle install`, create an issue in
```
Comment (segiddins) [inline source/v1.15/guides/rubygems_tls_ssl_troubleshooting_guide.html.md]: `If` should not be title cased

File: source/v1.15/guides/rubygems_tls_ssl_troubleshooting_guide.html.md [hunk 3]
```diff
@@ -26,312 +30,408 @@ If you're not interested in the reasons, and just want to get things fixed as qu
...
+the [Bundler issue tracker](https://github.com/bundler/bundler).)
 
-4. Which troubleshooting steps you have tried so far
+Please include:
+- Your Ruby version:
+- Your RubyGems version:
```
Comment (segiddins) [inline source/v1.15/guides/rubygems_tls_ssl_troubleshooting_guide.html.md]: `gem env`

File: source/v1.15/guides/rubygems_tls_ssl_troubleshooting_guide.html.md [hunk 4]
```diff
@@ -26,312 +30,408 @@ If you're not interested in the reasons, and just want to get things fixed as qu
...
+- Your Bundler version:
```
Comment (segiddins) [inline source/v1.15/guides/rubygems_tls_ssl_troubleshooting_guide.html.md]: `bundle env`

**Comment by indirect** [comment]
@bundlerbot r+

**Pull request merged**